plugins/cost-insights/src/components/CostOverviewCard/CostOverviewCard.tsx

```diff
@@ -59,22 +68,26 @@ export const CostOverviewCard = ({
     : null;
   const styles = useOverviewTabsStyles(theme);
 
+  const breakdownTabs = Object.keys(dailyCostData.groupedCosts ?? {}).map(
+    key => ({
+      id: key,
+      label: `Breakdown by ${key}`,
+      title: `Cloud Cost By ${key.charAt(0).toUpperCase() + key.slice(1)}`,
```

Comment:
Might be nice to move this out to its own `capitalize` utility function

Reply:
I discovered while moving it that MUI has a `capitalize` that does the exact same thing 😅

Reply:
> I discovered while moving it that MUI has a capitalize that does the exact same thing 😅

TIL